tests/unit/h/views/account_signup_test.py

```diff
@@ -47,12 +47,19 @@ def test_get(self, views, get_csrf_token, pyramid_request, feature_service):
                     "log_in_with_google": sentinel.google_enabled,
                     "log_in_with_facebook": sentinel.facebook_enabled,
                 },
+                "forOAuth": False,
                 "form": {},
                 "urls": {
                     "login": {
-                        provider: pyramid_request.route_url(f"oidc.login.{provider}")
-                        for provider in ("facebook", "google", "orcid")
-                    }
+                        "username_or_email": pyramid_request.route_url("login"),
+                        **{
+                            provider: pyramid_request.route_url(
+                                f"oidc.login.{provider}"
+                            )
+                            for provider in ("facebook", "google", "orcid")
+                        },
+                    },
+                    "signup": pyramid_request.route_url("signup"),
```

Comment:
There are several very similar changes, which tells me the tests should probably mock the code that generates these links and have just one test for them.